app/_components/Article/index.tsx

```
@@ -0,0 +1,37 @@
+      {data.thumbnail && (
+        <Image
+          src={data.thumbnail.url}
+          alt=""
+          className={styles.thumbnail}
+          width={data.thumbnail.width}
+          height={data.thumbnail.height}
+        />
+      )}
+      <div
+        className={styles.content}
+        dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)